api/types_alibabaprovider.go

```diff
@@ -0,0 +1,364 @@
+/*
```

Comment:
Do we need to include stuff that is outside the bounds of the placement group API?